src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/ListTablesIntegrationTest.java

```diff
@@ -0,0 +1,192 @@
+                                "tinyint_type": "tinyint",
+                                "uuid_type": "uuid",
+                                "varint_type": "varint",
+                                "map_type": {
+                                    "type": "map",
+                                    "keyType": "text",
+                                    "valueType": "int"
+                                },
+                                "list_type": {
+                                    "type": "list",
+                                    "valueType": "text"
+                                },
+                                "set_type": {
+                                    "type": "set",
+                                    "valueType": "text"
+                                },
+                                "vector_type": {
+                                    "type": "vector",
+                                    "dimension": 1024,
+                                    "service": {
+                                        "provider": "mistral",
+                                        "modelName": "mistral-embed"
+                                    }
+                                }
+                            },
+                            "primaryKey": {
+                                "partitionBy": [
+                                    "text_type"
+                                ],
+                                "partitionSort": {
+                                    "int_type": 1,
+                                    "bigint_type": -1
+                                }
+                            }
+                        },
+                        "options": {
+                            "ifNotExists": true
+                        }
+                    }
+                    """;
+    createTable(tableData);
+    String table2 =
+        """
+                    {
+                        "name": "person",
+                        "definition": {
+                            "columns": {
+                                "id": "text",
+                                "age": "int",
+                                "name": "text",
+                                "city": "text"
+                            },
+                            "primaryKey": "id"
+                        }
+                    }
+                """;
+    createTable(table2);
+  }
+
+  @Nested
+  @Order(1)
+  class ListTables {
+
+    @Test
+    public void listTablesOnly() {
+      String listTablesOnly =
+          """
+                    {}
+                    """;
+      listTables(listTablesOnly)
+          .hasNoErrors()
+          // Validate that status.tables is not null
+          .body("status.tables", notNullValue())
+
+          // Validate the number of tables in the response
+          .body("status.tables", hasSize(2))
+
+          // Validate the specific table names in any order
```

Comment:
This seems like incorrect statement? (tests that the first table is `allTypesTable`)